unittest/CMakeLists.txt

```diff
@@ -19,13 +19,14 @@ ENDIF(UNIX)
 
 
 SET(${PROJECT_NAME}_CPP_TESTS
-  test_states
-  test_actuation
+  # test_states
+  # test_actuation
```

Comment:
No!! If your unittest isn't passing, we need to either 
1) improve the unittest
2) correct your implementation

Please don't comment an existing test like this! This would affect all future PRs.